modules/prometheus-exporter_oracle/conf/readme.yaml

```diff
@@ -0,0 +1,2 @@
+documentations:
```

Comment:
link to prometheus exporter and perhaps also some oracle database monitoring ref

Reply:
I will do a simple doc on github with this prometheus exporter. But oracle monitoring ref are exclusively internal from my side.

Reply:
while all detectors on this module are usable with your "simple doc" no problem